src/bindings/python/src/pyopenvino/core/core.cpp

```diff
@@ -452,16 +449,14 @@ void regclass_Core(py::module m) {
            py::object model_path,
            py::object weights_path,
            const std::map<std::string, py::object>& config) {
-            if (py::isinstance(model_path, pybind11::module::import("io").attr("BytesIO"))) {
-                std::stringstream _stream;
+            if (py::isinstance(model_path, py::module::import("io").attr("BytesIO"))) {
```

Comment:
Out of curiosity, what's the difference between `pybind11::module::import("io").attr("BytesIO"))` and `py::module::import("io").attr("BytesIO"))`?

Reply:
there is no difference :)
We have in the begging of each file `namespace py = pybind11;`, so I just used `py` to be consistent with the rest of the file